sdk/src/types/api/core.rs

```diff
@@ -10,11 +10,13 @@ use alloc::{
 
 use serde::{Deserialize, Serialize};
 
+// Re-export types that are also used as responses.
+pub use crate::types::block::{slot::SlotCommitment as Commitment, Block as BlockResponse};
```

Comment:
I think my other comment got buried in the GH accordion so I'm bumping it: please remove the `Commitment` alias

Reply:
Done, also for `BlockResponse` and `OutputMetadataResponse` because your reasoning applies there, too.